awscli/testutils.py

```diff
@@ -57,6 +57,7 @@
 from awscli.plugin import load_plugins
 from awscli.clidriver import CLIDriver
 from awscli import EnvironmentVariables
+import awscli.errorhandler
```

Comment:
I would prefer we just disable the cli specific errorhandler for this client.  We eventually want to remove the `errorhandler` module entirely so new code shouldn't really be using it.